lua/lazyvim/plugins/extras/editor/bigfile.lua

```diff
@@ -0,0 +1,19 @@
+return {
+  "LunarVim/bigfile.nvim",
+  event = { "BufReadPre", "BufNewFile" },
+  opts = {
+    pattern = function(bufnr, filesize_mib)
+      if filesize_mib >= 2 then
+        return true
+      end
```

Comment:
```suggestion

```
This is already default and is used as fallback when the function returns `false`. Did you experience otherwise and decided to put this here? From `docs`
> You can add your own logic for detecting big files by setting pattern in the config to a function. If the function returns true file will be considered big, otherwise filesize will be used as a fallback

Reply:
Then you should use the `filesize` option, since it wouldn't be hardcoded in the pattern and the users can easily change it in their personal config in my personal opinion.

Reply:
thats a nice suggestion . I will do that